src/ui.text-alert/index.vue

```diff
@@ -1,30 +1,54 @@
 <template>
   <div v-if="isShownAlert" :data-cy="dataCy" v-bind="wrapperAttrs">
+    <!-- @slot Use it to add something above text. -->
+    <slot name="top" />
+
+    <div v-bind="titleAttrs">
+      <!-- @slot Use it to add something in the title. -->
+      <slot name="title" :title="title">
+        <div v-text="title" />
+      </slot>
+    </div>
+    <div v-bind="descriptionAttrs">
+      <!-- @slot Use it to add something in the description. -->
+      <slot name="description" :description="description">
+        <div v-text="description" />
```

Comment:
nested div no need, outer div `<div v-bind="descriptionAttrs" v-text="description">` should be inside of the slot